tempesta_fw/cache.c

```diff
@@ -1144,11 +1144,12 @@ cache_req_process_node(TfwHttpReq *req, unsigned long key,
 	TfwHttpResp *resp = NULL;
 	TDB *db = node_db();
 	TdbIter iter;
+	time_t is_live = tfw_cache_entry_is_live(req, ce);
```

Comment:
From its name, `is_live` appears to be a binary variable, but it's not. Perhaps, `lifetime` is a better name here.